### ISSUE TITLE
hdf5 does not compile on mac OSX

### ISSUE THREAD

**Issue by hervetan** [vbrankov/hdf5-ocaml#3]
Config: MAC OS X Sierra. Ocaml: 4.02.0 (also tried with 4.02.3)
Installed HDF5 lib by homebrew
Compilation error is:
ocaml setup.ml -build 
/Users/username/.opam/4.02.0/bin/ocamlopt.opt unix.cmxa -I /Users/username/.opam/4.02.0/lib/ocaml/ocamlbuild /Users/username/.opam/4.02.0/lib/ocaml/ocamlbuild/ocamlbuildlib.cmxa myocamlbuild.ml /Users/username/.opam/4.02.0/lib/ocaml/ocamlbuild/ocamlbuild.cmx -o myocamlbuild
/Users/username/.opam/4.02.0/bin/ocamlfind ocamlc -ccopt -I/usr/include/hdf5/serial -c lib/raw/h5a_stubs.c
mv h5a_stubs.o lib/raw/h5a_stubs.o
/Users/username/.opam/4.02.0/bin/ocamlfind ocamlc -ccopt -I/usr/include/hdf5/serial -c lib/raw/h5d_stubs.c
mv h5d_stubs.o lib/raw/h5d_stubs.o
/Users/username/.opam/4.02.0/bin/ocamlfind ocamlc -ccopt -I/usr/include/hdf5/serial -c lib/raw/h5f_stubs.c
+ /Users/username/.opam/4.02.0/bin/ocamlfind ocamlc -ccopt -I/usr/include/hdf5/serial -c lib/raw/h5f_stubs.c
lib/raw/h5f_stubs.c:123:8: error: no member named 'super_ext_size' in 'struct H5F_info2_t'
  info.super_ext_size = Int_val(Field(info_v, 0));
  ~~~~ ^
lib/raw/h5f_stubs.c:135:40: error: no member named 'super_ext_size' in 'struct H5F_info2_t'
  Store_field(info_v, 0, Val_int(info->super_ext_size));
                                 ~~~~  ^
/Users/username/.opam/4.02.0/lib/ocaml/caml/mlvalues.h:75:29: note: expanded from macro 'Val_int'
#define Val_int(x) Val_long(x)
                            ^
/Users/username/.opam/4.02.0/lib/ocaml/caml/mlvalues.h:71:36: note: expanded from macro 'Val_long'
#define Val_long(x)     (((intnat)(x) << 1) + 1)
                                   ^
/Users/username/.opam/4.02.0/lib/ocaml/caml/memory.h:239:27: note: expanded from macro 'Store_field'
  value caml__temp_val = (val); \
                          ^~~
2 errors generated.
Command exited with code 2.

**Comment by markghayden** [comment]
I'm running into same problem.  Any resolution to this?

**Comment by vbrankov** [comment]
This should be fixed in the just released version 0.1.3.  It is in the OPAM pipeline and I hope it will be through by tomorrow.

**Comment by struktured** [comment]
@vbrankov this can be closed
